target_frame: velodyne # Leave disabled to output scan in pointcloud frame
transform_tolerance: 0.01
min_height: -1.5 # default = -0.8 / KIAPI = -1.5
max_height: -1.0 # default = 0.3 / KIAPI= -0.9
angle_min: -3.1408 # -M_PI/2
angle_max: 3.1408 # M_PI/2
angle_increment: 0.0007 # M_PI/360.0
scan_time: 0.0333
range_min: 0.0
range_max: 50.0
use_inf: true
inf_epsilon: 100.0

# Concurrency level, affects number of pointclouds queued for processing and number of threads used
# 0 : Detect number of cores
# 1 : Single threaded
# 2->inf : Parallelism level
concurrency_level: 3

map: {
  use_ransac: false,
  pcl_lx: 60,
  pcl_ly: 20,
  pcl_z_min: -1.6,
  pcl_dbscan_minpnts: 3,
  pcl_dbscan_eps: 0.3,
  ransac_post_inclusion_offset: 0.3,
  ransac_distance_threshold: 0.2
}

global_planner: {
  car_width: 1.0,
  grid_resolution: 0.3,
  cut_tail_by_angle: true,
  max_steering_angle: 0.08722,
  use_line_smoothing_from_current_position: false,
  smoothing_distance: 10.0,
  smoothing_cliff_min_bias: 0.5,
  smoothing_cliff_ratio: 0.5,
  smoothing_summit_ratio: 0.5,
  start_smoothing_distance: 6.0,
  corridor_width_min: 0.5,
  corridor_width_dynamic_min: 8.0,
  corridor_max_bias: 100.0,
  safe_distance: 6,
  nominal_acceleration: 0.5,
  object_velocity_threshold: 0.1,
  max_obstacle_prediction_query_size: 1000,
  acc_stop_distance: 5,
  acc_stop_angle: 0.08722,
  use_static_object: false,
  use_lane_point_first: false,
  smoothing_cliff_n_check_idx: 10,
  blocked_by_object_distance: 1.0,
  blocked_by_object_inflation_distance: 3.0
}

use_keti_velocity: true
use_nominal_obstacle_rad: false
lane_width: 3